### ISSUE TITLE
Question: How to learn IP address assigned when using IPAM for TransportServer/VirtualServer CR

### ISSUE THREAD

**Issue by bukovjanmic** [F5Networks/k8s-bigip-ctlr#1916]
#### Title
Unable to determine IP address assigned for TransportServer

#### Description
We are trying out IPAM support for TransportServer service. The IP allocation works and the virtual server on F5 gets created, but we are unable to see which IP address got assigned in the TransportServer CR.

We can parse the F5IPAM object in kube-system, but that would require to give permission to this object to everyone who is able to create TransportServer CR and then they would see IP addresses and other information from TransportServers from other projects/namespaces.

#### Actual Problem
We see no clear place where to learn assigned IP address when using IPAM and TransportServer CR, from the tenant position (e.g. TransportServer is create in a namespace and the creator does not have access to kube-system namespace).

#### Solution Proposed
Possibly write out a status: section to TransportServer CR once an IP address is successfully allocated through IPAM so that the owner of the CR can use it in subsequent actions.

#### Alternatives
Any other place in the namespace where TransportServer is created (Service? ConfigMap? Secret?)

**Comment by trinaths** [comment]
@bukovjanmic can you try, `kubectl get ts -o wide`

**Comment by bukovjanmic** [comment]
Sure:
```
[~]$ oc get ts -o wide
NAME                          VIRTUALSERVERADDRESS   VIRTUALSERVERPORT   POOL        POOLPORT   AGE
pgadmin-virtual-server-ipam                          80                  pgadminf5   5050       120m
psql1-virtual-server          172.31.8.226           5432                pgf5        5432       78m
```

I have two TraspoertServers, one using ipam and one has the address specified manually.
The IPAM version does not contain the address anywhere.

Note that it actually works, the virtual address is created and the address from IPAM range is correctly assigned, the only problem is that the IP address is nowhere to be found in the namespace where TrasportServer is created, only in kube-system F5IPAM CR where the user does not normally have access.

**Comment by trinaths** [comment]
Is that with IPAM static approach or InfoBlox ?
________________________________

**Comment by bukovjanmic** [comment]
Not sure, probably static approach. We follow this: https://clouddocs.f5.com/containers/latest/userguide/ipam/

We have deployd an ipam controller in kube-system like this, there is a static range defined:

kind: Deployment
apiVersion: apps/v1
metadata:
  annotations:
    deployment.kubernetes.io/revision: '1'
  name: f5-ipam-controller
  namespace: kube-system
  labels:
    name: f5-ipam-controller
spec:
  replicas: 1
  selector:
    matchLabels:
      app: f5-ipam-controller
  template:
    metadata:
      creationTimestamp: null
      labels:
        app: f5-ipam-controller
    spec:
      containers:
        - name: f5-ipam-controller
          image: 'f5networks/f5-ipam-controller:latest'
          command:
            - /app/bin/f5-ipam-controller
          args:
            - '--orchestration=kubernetes'
            - '--ip-range=''{"Test":"172.31.8.230-172.31.8.240"}'''
            - '--log-level=DEBUG'
          resources: {}
          terminationMessagePath: /dev/termination-log
          terminationMessagePolicy: File
          imagePullPolicy: IfNotPresent
      restartPolicy: Always
      terminationGracePeriodSeconds: 30
      dnsPolicy: ClusterFirst
      serviceAccountName: ipam-ctlr
      serviceAccount: ipam-ctlr
      securityContext: {}
      schedulerName: default-scheduler
  strategy:
    type: RollingUpdate
    rollingUpdate:
      maxUnavailable: 25%
      maxSurge: 25%
  revisionHistoryLimit: 10
  progressDeadlineSeconds: 600

**Comment by trinaths** [comment]
@bukovjanmic - Let me investigate and update here.

**Comment by trinaths** [comment]
Created CONTCNTR-2847 for internal PM tracking

**Comment by sravyap135** [comment]
Fix will be part of 2.7.0 CIS release
